compiler/kernel/op.go

```diff
@@ -57,7 +58,8 @@ type Builder struct {
 	progress  *zbuf.Progress
 	arena     *zed.Arena // For zed.Values created during compilation.
 	deletes   *sync.Map
-	funcs     map[string]expr.Function
+	udfs      map[string]dag.Expr
+	udfStack  map[string]*expr.UDF
```

Comment:
`udfStack` feels like a misleading name for this since it doesn't behave like a stack.

I admit this isn't great but maybe have `udfDefs map[string]dag.Expr` and `udfs map[string]*expr.UDF` since the first stores definitions and the second, instances?

Reply:
How about s/udfStack/compiledUDFs ?